app/react/package.json

```diff
@@ -33,12 +33,15 @@
     "@babel/runtime": "^7.1.2",
     "@emotion/styled": "^0.10.6",
     "@storybook/core": "4.0.0-rc.1",
+    "@storybook/node-logger": "^3.4.11",
     "babel-plugin-react-docgen": "^2.0.0",
     "common-tags": "^1.8.0",
     "global": "^4.3.2",
     "lodash": "^4.17.11",
     "prop-types": "^15.6.2",
-    "react-dev-utils": "^6.0.5"
+    "react-dev-utils": "^6.0.5",
+    "semver": "^5.6.0",
+    "webpack": "^4.21.0"
```

Comment:
webpack is a direct dep in a bunch of the SB packages (including `core`). It will be complicated to make it a peer (like for supporting different major version of webpack) 

CC #4371, #4430